example_agents/sb3_agent/components.yaml

```diff
@@ -5,20 +5,21 @@ repos:
 
 
 components:
-    agent:
+    sb3_agent:
```

Comment:
I had changed it because when working with multiple example_agents in various tests, it was a little confusing to have all of the example agents use the same name for their components. It also was an opportunity to exercise the code path where the agent component being run has a different name than the default (and thus the user is required to specify the agent name explicitly).

I don't feel strongly about it. Happy to change it back to `agent` if you have a preference. For now I'll update the README.